test/system/runScenario3.js

```diff
@@ -0,0 +1,18 @@
+const runScenario3 = (trader2) => {
+  console.log('********************* Scenario 3 :  Test balance checks *********************');
+  return trader2.placeLimitOrder('S', trader2.getBalance().assets + 25, 0.14, { expectFail: true })
+    .then(() => trader2.placeLimitOrder('S', ((trader2.getBalance().assets/2)), 0.14))
+    .then(() => trader2.placeLimitOrder('S', ((trader2.getBalance().assets * 2) * 3 / 4), 0.14, { expectFail: true }))
+    .then(() => trader2.cancelAllOrders())
+    .then(() => trader2.getCurrentBalance(trader2.config))
```

Comment:
Is there a good reason we pass trader2's own config to trader2 for this call?
Would we ever pass anything else? If not, then we should remove that argument.